libpod/image/utils.go

```diff
@@ -174,3 +182,63 @@ func imageNameForSaveDestination(img *Image, imgUserInput string) string {
 	}
 	return fmt.Sprintf("%s%s", prepend, imgUserInput)
 }
+
+func retryIfNecessary(operation func() error) error {
```

Comment:
Can you put this code into containers/common, since podman, Buildah and Skopeo are all going to use it?

Reply:
This functionality along with @ParkerVR pulling of shortnames should be put into common/pkg/image  shared functions.

Reply:
I can draft a PR in c/common, will this be common/pkg/image/image.go or pull.go? @ParkerVR?

Reply:
I have been testing locally with pkg/image/pull.go, which I think makes more sense for consistency.

Reply:
containers/common PR has been merged.